src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,98 @@
+.card-container {
+  max-width: $card-container-width;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  padding: 50px 0;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:not(:nth-of-type(4n)) {
+    margin-right: $product-card-margin;
```

Comment:
don't give margins to BEM-block. Use BEM-element class for that (e.g. catalog__card)